File: 1672-richest-customer-wealth/1672-richest-customer-wealth.c


int maximumWealth(int** accounts, int accountsSize, int* accountsColSize){
    
    
    
        int res = 0;
        for(int i =0;i<accountsSize;i++){
            int temp = 0;
            for(int j = 0;j<*accountsColSize;j++){
                temp+=accounts[i][j];
            }
            if(temp > res) res = temp;
        }
        return res;
}

File: 1672-richest-customer-wealth/1672-richest-customer-wealth.java
class Solution {
    public int maximumWealth(int[][] accounts) {
        int maxVealth = 0;
        for (int i = 0; i <accounts.length;i++){
            int total = 0;
            for (int j = 0; j < accounts[i].length;j++){
                total += accounts[i][j];
            }
            if (total > maxVealth){
                maxVealth = total;
            }
        }
        return maxVealth; 
    }
}